mantidimaging/gui/windows/image_load_dialog/test/presenter_test.py

```diff
@@ -24,17 +27,17 @@ def test_do_update_sample_with_no_selected_file(self):
 
         self.v.select_file.assert_called_once_with("Sample")
 
-    @mock.patch("mantidimaging.gui.windows.image_load_dialog.presenter.find_log", return_value=3)
+    @mock.patch("mantidimaging.gui.windows.image_load_dialog.presenter.find_log_for_image", return_value=3)
     @mock.patch("mantidimaging.gui.windows.image_load_dialog.presenter.find_180deg_proj", return_value=2)
     @mock.patch("mantidimaging.gui.windows.image_load_dialog.presenter.find_images", return_value=1)
     @mock.patch("mantidimaging.gui.windows.image_load_dialog.presenter.load_log")
     @mock.patch("mantidimaging.gui.windows.image_load_dialog.presenter.read_in_file_information",
-                return_value=FileInformation([], (0, 0, 0), True))
+                return_value=FileInformation(["a"], (0, 0, 0), True))
     @mock.patch(
         "mantidimaging.gui.windows.image_load_dialog.presenter.get_file_extension", )
     @mock.patch("mantidimaging.gui.windows.image_load_dialog.presenter.get_prefix")
     def test_do_update_sample(self, get_prefix, get_file_extension, read_in_file_information, mock_load_log,
```

Comment:
Not for now as it would be scope creep, but I think this test could be broken up into more compartmentalised tests making it easier to understand failures when and if they occur

Reply:
I'm hoping this will also get better with further cleanups